cabal.project

```diff
@@ -37,15 +37,17 @@ optional-packages: vendored/*/*.cabal
 tests: true
 
 package *
-  ghc-options: -haddock
+  -- ghc 8.10 cannot build ghc-lib 9.2 with --haddock
```

Comment:
I'm still confused by this comment.

Reply:
ghc-lib 9.2 contains illegal haddock markup which triggers a build error in ghc 8.10. This was demoted to a warning in ghc 9.x

This is only a problem now because previously we fixed hlint to v3.2 when building ghc 8.10.7 and this problem doesn't arise